pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -19,6 +19,10 @@ type ClusterAutoscalerSpec struct {
 	// Gives pods graceful termination time before scaling down
 	MaxPodGracePeriod *int32 `json:"maxPodGracePeriod,omitempty"`
 
+	// Maximum time CA waits for node to be provisioned
+	// +kubebuilder:validation:Pattern=([0-9]*(\.[0-9]*)?[a-z]+)+
```

Comment:
This one needs to match as well else you will overwrite the pattern when we run `make gen-crd` next
```suggestion
	// +kubebuilder:validation:Pattern=^([0-9]+(\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$
```

Reply:
Ah yeah I missed that - thanks!